diff --git a/build/useShareIntent.js b/build/useShareIntent.js
index 999c40840b464496f1755a911f544f57ee20f048..35eb5d7c2a9d9afb5146e6d6ae12df04797020ff 100644
--- a/build/useShareIntent.js
+++ b/build/useShareIntent.js
@@ -27,6 +27,10 @@ export default function useShareIntent(options = SHAREINTENT_OPTIONS_DEFAULT) {
      * Call native module on universal linking url change
      */
     const refreshShareIntent = () => {
+        if (!isReady) {
+            options.debug && console.debug("useShareIntent[refresh] skip until listeners are ready");
+            return;
+        }
         options.debug && console.debug("useShareIntent[refresh]", url);
         if (url?.includes(`${getScheme(options)}://dataUrl=`)) {
             // iOS only
@@ -47,7 +51,7 @@ export default function useShareIntent(options = SHAREINTENT_OPTIONS_DEFAULT) {
         options.debug &&
             console.debug("useShareIntent[mount]", getScheme(options), options);
         refreshShareIntent();
-    }, [url, options.disabled]);
+    }, [url, options.disabled, isReady]);
     /**
      * Handle application state (active, background, inactive)
      */
